### PR TITLE
 Translate 2019-03-05-multiple-vulnerabilities-in-rubygems.md (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Translate e2f18e4

diff for review is 11c433a
Could you review this? @ruby/www-ruby-lang-org-i18n-ko

### CODE REVIEW

File: ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## 해당 버전
+
+* 루비 2.4 버전대: 2.4.5 이하
+* 루비 2.5 버전대: 2.5.3 이하
+* 루비 2.6 버전대: 2.6.1 이하
+* 리비전 67168 이전의 트렁크
+
+## 해결 방법
+
+RubyGems 2.7.6.2/2.7.9/3.0.3 이나 그 이후의 버전은 취약점에 대한 패치를 포함하고 있으므로, RubyGems를 최신 버전으로 업그레이드해 주세요.
```
Comment (riseshia) [inline ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md]: https://github.com/ruby/www.ruby-lang.org/pull/2002/files#diff-c7028eca1eb99d89959953d676dda854R36 included

File: ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## 세부 내용
+
+아래와 같은 취약점이 보고되었습니다.
+
+* CVE-2019-8320: tar 압축 풀기 중 심볼릭 링크를 이용한 폴더 삭제
+* CVE-2019-8321: `verbose`에서의 이스케이프 시퀀스 주입 취약점
+* CVE-2019-8322: `gem owner`에서의 이스케이프 시퀀스 주입 취약점
+* CVE-2019-8323: API 응답 처리 중의 이스케이프 시퀀스 주입 취약점
+* CVE-2019-8324: 임의의 코드 실행을 유도할 수 있는 악의있는 젬 설치
```
Comment (yous) [inline ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md]: How about '악의적인' or '악의 있는'?
```suggestion
* CVE-2019-8324: 임의의 코드 실행을 유도할 수 있는 악의적인 젬 설치
```

File: ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md (hunk 1 above)
Comment (yous) [inline ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md]:
```suggestion
RubyGems 2.7.6.2/2.7.9/3.0.3이나 그 이후의 버전은 취약점에 대한 패치를 포함하고 있으므로, RubyGems를 최신 버전으로 업그레이드해 주세요.
```

File: ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+* CVE-2019-8325: 에러 중 이스케이프 시퀀스 주입 취약점
+
+루비 사용자는 가능한 빨리 업그레이드하거나 아래 해결 방법으로 조치하기 바랍니다.
```
Comment (yous) [inline ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md]: Upgrading is also mentioned in Workarounds section, so how about "루비 사용자는 가능한 빨리 아래 해결 방법으로 조치하기 바랍니다."?

File: ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+gem update --system
+```
+
+RubyGems를 업그레이드를 할 수 없다면, 차선책으로 다음 패치를 적용할 수 있습니다.
+
+* [루비 2.4.5](https://bugs.ruby-lang.org/attachments/7669)
+* [루비 2.5.3](https://bugs.ruby-lang.org/attachments/7670)
+* [루비 2.6.1](https://bugs.ruby-lang.org/attachments/7671)
+
+개발버전에 대하여는, 최신 리비전으로 업데이트합니다.
```
Comment (yous) [inline ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md]:
```suggestion
개발 버전에 대하여는, 최신 리비전으로 업데이트합니다.
```

File: ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md (hunk 3 above)
Comment (riseshia) [reply ko/news/_posts/2019-03-05-multiple-vulnerabilities-in-rubygems.md]: 👍  fixed 889cb3f2

**Pull request merged**